### ISSUE TITLE
[BUG REPORT] CrewLink doensn't work between IPv4 and IPv6

### ISSUE THREAD

**Issue by MoritzLerch** [ottomated/CrewLink#501]
We experienced problems between IPv4 and IPv6 internet connections. This bug makes **players with an IPv4 only able to speak to other IPv4-users** and **IPv6-players can only talk to IPv6-guys**. 

So in a lobby you can only talk to people with the same IP-version. Maybe someone here has more clue on this problem. 

Maybe thats an issue which I should better post in the [server-repo](https://github.com/ottomated/CrewLink-server) but I'll post it here so more people can maybe comment to this.

**Comment by Stogoh** [comment]
Duplicate. See: #271 

We've disabled IPv6 on all computers.

**Comment by MoritzLerch** [comment]
But is there a better solution than using a VPN? Only being able to talk to the other ones through a VPN is kind of annoying. I don't know but maybe someone has another idea...

**Comment by Stogoh** [comment]
Yes. As I said just disabled IPv6 in the network settings on all attending computers. That's pretty much it. No need for a VPN at all.

![image](https://user-images.githubusercontent.com/2563102/103543855-7d118980-4e9f-11eb-89a0-eaa2199dde19.png)

**Comment by MoritzLerch** [comment]
Oh nice! You're German as well.

**Comment by itsTyrion** [comment]
Please re-open, disabling v6 is NOT a solution. It's a hack, for now, but not a solution.

**Comment by edoriontlg** [comment]
This fix isn't working, we disabled IPv6 and it's still not working

**Comment by itsTyrion** [comment]
> This fix isn't working, we disabled IPv6 and it's still not working

Yep. You could try using a vpn (hide.me is my free VPN of choice)

**Comment by Classist** [comment]
This is a big problem to use the program, the workaround is very complicated for people who have no computer experience. And we can't use older versions because the autoupdate feature. Should be fixed ASAP.

**Comment by billyjbryant** [comment]
Disabling IPv6 or using a VPN is a bandaid, not a resolution. Ideally, if Crewlink finds that there are more than one IP and interface is detected, a dropdown would allow the user to select the correct interface and IP address to bind the Crewlink app to.

**Comment by itsTyrion** [comment]
Exactly. Or it could just like.. translate? 🤔

**Comment by Muceule** [comment]
Please use IPv6 instead of IPv4 in Crewlink...
Reason for "no hearing" anyone, is, that a lot of people has DS-Lite only and no direct IPv4 adress anymore. Crewlink try to create a ipv4 linkage... 
Same problem with XBox Voice Chat and Nintendo Switch multiplayer.

**Comment by itsTyrion** [comment]
> 
> 
> Please use IPv6 instead of IPv4 in Crewlink...
> Reason for "no hearing" anyone, is, that a lot of people has DS-Lite only and no direct IPv4 adress anymore. Crewlink try to create a ipv4 linkage...
> Same problem with XBox Voice Chat and Nintendo Switch multiplayer.

Not an option, there are still people without a v6

**Comment by Muceule** [comment]
> > Please use IPv6 instead of IPv4 in Crewlink...
> > Reason for "no hearing" anyone, is, that a lot of people has DS-Lite only and no direct IPv4 adress anymore. Crewlink try to create a ipv4 linkage...
> > Same problem with XBox Voice Chat and Nintendo Switch multiplayer.
> 
> Not an option, there are still people without a v6

First, that's really bad, because IPv4 adress are too limited and not everyone has one. 
Then the crewlink server shall redirect all data to the right IPv4 or IPv6 address.
